app/services/audit/catalog_to_moab.rb

```diff
@@ -49,6 +36,36 @@ def results
 
     private
 
+    def druid
+      @druid ||= complete_moab.preserved_object.druid
+    end
+
+    def check_preserved_object_and_complete_moab_versions_match
```

Comment:
The intent of calling it "check_*" instead of "*?" is to indicate that there is more happening than just an equality check. I'm open to other suggestions for a follow-up PR.